model/src/w3profsmd_pdlib.F90

```diff
@@ -5585,1055 +5529,648 @@ SUBROUTINE PDLIB_JACOBI_GAUSS_SEIDEL_BLOCK(IMOD, FACX, FACY, DTG, VGX, VGY)
+#ifdef W3_ST4
+    USE W3SRC4MD, only: W3SPR4
+#endif
+#ifdef W3_REF1
+      USE W3GDATMD, only: REFPARS
+#endif
+#ifdef W3_MEMCHECK
+    USE MallocInfo_m
+#endif
+    implicit none
+    LOGICAL, INTENT(IN) :: LCALC 
     INTEGER, INTENT(IN) :: IMOD
     REAL, INTENT(IN) :: FACX, FACY, DTG, VGX, VGY
     !
-    INTEGER :: IP, ISP, ITH, IK, JSEA, ISEA, IP_glob
+    INTEGER :: IP, ISP, ITH, IK, JSEA, ISEA, IP_glob, IS0
     INTEGER :: myrank
     INTEGER :: nbIter, ISPnextDir, ISPprevDir
     INTEGER :: ISPp1, ISPm1, JP, ICOUNT1, ICOUNT2
     ! for the exchange
-    REAL  :: CCOS, CSIN, CCURX, CCURY
-    REAL  :: eSum(NSPEC)
-    REAL  :: eA_THE, eC_THE, eA_SIG, eC_SIG, eSI
-    REAL  :: CAD(NSPEC), CAS(NSPEC), ACLOC(NSPEC)
-    REAL  :: CP_SIG(NSPEC), CM_SIG(NSPEC)
-    REAL  :: eFactM1, eFactP1
+    REAL*8  :: CCOS, CSIN, CCURX, CCURY
+    REAL*8  :: eSum(NSPEC), FRLOCAL
+    REAL*8  :: eA_THE, eC_THE, eA_SIG, eC_SIG, eSI
+    REAL*8  :: CAD(NSPEC), CAS(NSPEC), ACLOC(NSPEC)
+    REAL*8  :: CP_SIG(NSPEC), CM_SIG(NSPEC)
+    REAL*8  :: eFactM1, eFactP1
     REAL*8  :: Sum_Prev, Sum_New, p_is_converged, DiffNew, prop_conv
-    REAL  :: Sum_L2, Sum_L2_GL
-    REAL  :: DMM(0:NK2)
-    REAL  :: eDiff(NSPEC), eProd(NSPEC)
-    REAL  :: DWNI_M2(NK), CWNB_M2(1-NTH:NSPEC)
-    REAL  :: VAnew(NSPEC), VFLWN(1-NTH:NSPEC)
+    REAL*8  :: Sum_L2, Sum_L2_GL
+    REAL  :: DMM(0:NK2), DAM(NSPEC), DAM2(NSPEC), SPEC(NSPEC)
+    REAL*8  :: eDiff(NSPEC), eProd(NSPEC), eDiffB(NSPEC)
+    REAL*8  :: DWNI_M2(NK), CWNB_M2(1-NTH:NSPEC)
+    REAL  :: VAnew(NSPEC), VFLWN(1-NTH:NSPEC), JAC, JAC2
     REAL  :: VAAnew(1-NTH:NSPEC+NTH), VAAacloc(1-NTH:NSPEC+NTH)
-    REAL  :: VAinput(NSPEC), VAacloc(NSPEC), eDiffB(NSPEC),ASPAR_DIAG(NSPEC)
-    REAL  :: aspar_diag_local(nspec), aspar_off_diag_local(nspec), b_jac_local(nspec)
-    REAL  :: eDiffSing, eSumPart
-    REAL  :: eVal1, eVal2!, extmp(nspec,nseal)
-    REAL  :: eVA, CG2
-    REAL  :: CG1(0:NK+1), WN1(0:NK+1)
-    LOGICAL :: LCONVERGED(NSEAL), lexist
-#ifdef WEIGHTS
-    INTEGER :: ipiter(nseal), ipitergl(np_global), ipiterout(np_global)
-#endif
-#ifdef W3_DEBUGSRC
-    REAL :: IntDiff, eVA_w3srce, eVAsolve, SumACout
-    REAL :: SumVAin, SumVAout, SumVAw3srce, SumVS, SumVD, VS_w3srce
-    REAL    :: VAsolve(NSPEC)
-    REAL*8  :: ACsolve
-    REAL    :: eB
-#endif
-#ifdef W3_DEBUGSOLVERCOH
-    REAL :: TheARR(NSPEC, npa)
-    REAL :: PRE_VA(NSPEC, npa)
-    REAL :: OffDIAG(NSPEC, npa)
-    REAL*8 :: eOff(NSPEC)
-    REAL*8 :: eSum1(NSPEC), eSum2(NSPEC)
-#endif
-    CHARACTER(len=128) eFile
-    INTEGER ierr, i
-    INTEGER JP_glob
-    INTEGER is_converged, itmp
-
-    integer :: testknoten = 923
-
-    LOGICAL :: LSIG = .FALSE.
-
-    memunit = 50000+IAPROC
-    !AR: this is missing in init ... but there is a design error in ww3_grid with FLCUR and FLLEV
-    LSIG = FLCUR .OR. FLLEV
-#ifdef W3_DEBUGSOLVERCOH
-    OffDIAG = ZERO
-#endif
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 0')
```

Comment:
@aronroland, was this removed intentionally?

Reply:
@aronroland, I do see this now, at line 5603.  My apologies if it's always been there.  cleared.